data/json/monsters/mammal.json

```diff
@@ -1799,6 +1816,7 @@
     "dodge": 6,
     "vision_night": 5,
     "harvest": "mammal_small_fur",
+    "dissect": "dissect_feline_sample_single",
```

Comment:
Chose to split the foxes in half, according to behavior patterns.  This would be the one I change, but I worry about lack of feline samples in comparison to lupine.